### DISCUSSION TITLE
Dulicate Error Mongoose Schema on Post Request

### DISCUSSION THREAD

**Discussion by Gblur** [neuefische/questions#570] [Web] [answered]
Error: {"error":"E11000 duplicate key error collection: test.maps index: map.nodes.id_1 dup key: { map.nodes.id: null }"}

Reproduce: 
When I submit my Form Data and already have an entry in the Database I got an this error

Mongoose Schema: 
[Schema File](https://github.com/Gblur/capstone-project/blob/6805cbd90b8758cdc34dbab4a844f74ce774b338/db/models/Map.js)

function for returning Arrays of nodes and edges which get a unique ID by uuid()
and the Form itself

[Initial Nodes and Submit Function](https://github.com/Gblur/capstone-project/blob/6805cbd90b8758cdc34dbab4a844f74ce774b338/components/Form/index.js)

**Comment by PReichetanz** [comment]
Hi @Gblur,

can you show us the content of the document you've posted in the first place? (Cannot test it myself right now)

I'd especially like to see how the 'id' values look like as soon as they get into the database, but the entire document will help us as well.

**Reply by Gblur** [reply, marked as answer]
Nevermind 
It was a problem with the mongo atlas  Database.
I totally cleaned it by dropping the collection.
